src/components/file-component/FileComponent.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react'
+
+import { format } from 'date-fns'
```

Comment:
It is very interesting.
Do you connect a new library?
We don't use such a library and also I don't see it in package.json

Reply:
> It is very interesting. Do you connect a new library? We don't use such a library and also I don't see it in package.json

nope, we have it. Here it is: https://github.com/ita-social-projects/SpaceToStudy-Client/blob/c43206027d4995c997350219fbb7fd245b8200d0/package.json#L6